newIDE/app/src/EventsBasedObjectEditor/EventBasedObjectChildrenEditor.js

```diff
@@ -337,6 +337,8 @@ export default class EventBasedObjectChildrenEditor extends React.Component<
                 onUpdateBehaviorsSharedData={() =>
                   this.updateBehaviorsSharedData()
                 }
+                // TODO EBO Go to the behavior extension tab.
```

Comment:
This is for a next PR?

Reply:
Yes, it needs to a callback all the way to MainFrame. I didn't do it for functions either. I don't want to spend time on UI refinements until EBO editor is usable.